src/array.cr

```diff
@@ -862,11 +862,30 @@ class Array(T)
     self
   end
 
+  # Select array elements that meets conditions from `self`. Returns `self`.
```

Comment:
Likewise, how about `Removes all elements from `self` for which the given block returns falsy value."?